src/ui/inputs/BaseInputContainer.tsx

```diff
@@ -44,9 +44,10 @@ export const BaseInputContainer = ({
         {leftAdornment && <div>{leftAdornment}</div>}
         {cloneElement(children, {
           className:
-            'flex-1 truncate bg-input-bg text-text-input outline-none placeholder:text-text-input-placeholder disabled:text-text-input-disabled',
+            (children.props.className ?? '') +
+            ' min-h-7 min-w-0 flex-1 truncate bg-input-bg text-text-input outline-none placeholder:text-text-input-placeholder disabled:text-text-input-disabled',
         })}
-        {rightAdornment && <div className="min-w-0">{rightAdornment}</div>}
```

Comment:
I've added min-w since some buttons would actually overflow the BaseInputContainer's border. Could you please verify that blinks with large buttons would truncate and not overflow. I'll try finding an example for you to test.

Reply:
nice catch, thanks